app/src/main/java/org/mifos/mobile/ui/update_password/UpdatePasswordScreen.kt

```diff
@@ -0,0 +1,307 @@
+import androidx.compose.material3.ButtonDefaults
+import androidx.compose.material3.Icon
+import androidx.compose.material3.IconButton
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.SnackbarDuration
+import androidx.compose.material3.SnackbarHost
+import androidx.compose.material3.SnackbarHostState
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.saveable.rememberSaveable
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.ExperimentalComposeUiApi
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.platform.LocalSoftwareKeyboardController
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.input.KeyboardType
+import androidx.compose.ui.text.input.PasswordVisualTransformation
+import androidx.compose.ui.text.input.TextFieldValue
+import androidx.compose.ui.text.input.VisualTransformation
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.tooling.preview.PreviewParameter
+import androidx.compose.ui.unit.dp
+import androidx.hilt.navigation.compose.hiltViewModel
+import androidx.lifecycle.compose.collectAsStateWithLifecycle
+import org.mifos.mobile.R
+import org.mifos.mobile.core.ui.component.MifosOutlinedTextField
+import org.mifos.mobile.core.ui.component.MifosProgressIndicator
+import org.mifos.mobile.core.ui.component.MifosTopBar
+import org.mifos.mobile.core.ui.theme.MifosMobileTheme
+import org.mifos.mobile.ui.savings_account_withdraw.UiStatesParameterProvider
+import org.mifos.mobile.utils.Network
+import org.mifos.mobile.utils.RegistrationUiState
+
+@OptIn(ExperimentalComposeUiApi::class)
+@Composable
+fun UpdatePasswordScreen(
+    viewModel: UpdatePasswordViewModel = hiltViewModel(),
+    getBackToPreviousScreen: () -> Unit
+) {
+    var newPassword by rememberSaveable(stateSaver = TextFieldValue.Saver) {
+        mutableStateOf(TextFieldValue(""))
+    }
+    var confirmPassword by rememberSaveable(stateSaver = TextFieldValue.Saver) {
+        mutableStateOf(TextFieldValue(""))
+    }
+    var viewNewPassword by rememberSaveable {
+        mutableStateOf(false)
+    }
+    var viewConfirmPassword by rememberSaveable {
+        mutableStateOf(false)
+    }
+    var passwordMatches by rememberSaveable {
+        mutableStateOf(false)
+    }
+
+    var newPasswordError by rememberSaveable { mutableStateOf(false) }
+    var confirmPasswordError by rememberSaveable { mutableStateOf(false) }
+
+    var newPasswordErrorContent by rememberSaveable {
+        mutableStateOf("")
+    }
+    var confirmPasswordErrorContent by rememberSaveable {
+        mutableStateOf("")
+    }
```

Comment:
Can you make all these variable to have consistent look?
Means make all to be only in one line, similarly do for all the variable initialisations.